yky-social/src/YKYOperations.ts

```diff
@@ -469,6 +469,7 @@ static async mediaUpload(ctx: WorkflowContext, mtype: string, mediaId: string, m
         // No need to make a database record, or, at this point, roll anything back.
         // It might be a good idea to clobber the s3 key in case it arrived but we weren't told.
         //   (The access key duration is less than the time we wait, so it can't be started.)
+        // TODO: perhaps put this operation in a communicator later.
         await this.ensureS3FileDropped(ctx, mediaFile, bucket);
```

Comment:
Pre-existing really real bug... there's no 'this' in a static method, as written this will fail if it gets here.

Reply:
Resolved - replaced `this` with `Operations`